src/components/TokenBar.js

```diff
@@ -5,220 +5,204 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import { t } from 'ttag';
-import { connect } from "react-redux";
+import { useSelector, useDispatch } from 'react-redux';
 import { selectToken } from '../actions/index';
 import { get } from 'lodash';
 import hathorLib from '@hathor/wallet-lib';
 import helpers from '../utils/helpers';
 import wallet from "../utils/wallet";
 import Loading from '../components/Loading';
 import { TOKEN_DOWNLOAD_STATUS } from '../sagas/tokens';
-
-
-const mapStateToProps = (state) => {
-  return {
-    registeredTokens: state.tokens,
-    allTokens: state.allTokens,
-    selectedToken: state.selectedToken,
-    tokensBalance: state.tokensBalance,
-    tokenMetadata: state.tokenMetadata,
-  };
-};
-
-
-const mapDispatchToProps = dispatch => {
-  return {
-    selectToken: data => dispatch(selectToken(data)),
-  };
-};
-
-
-/**
- * Component that shows the left bar of tokens
- *
- * @memberof Components
- */
-class TokenBar extends React.Component {
-  constructor(props) {
-    super(props);
-
-    /**
-     * opened {boolean} If bar is opened or not (default is false)
-     * selected {string} Symbol of the selected token (default is 'HTR')
-     */
-    this.state = {
-      opened: false,
-      selected: 'HTR',
-    }
+import { useHistory, useLocation } from 'react-router-dom';
+
+// Routes that should display the TokenBar
+const ROUTE_WHITELIST = [
+  '/unknown_tokens/',
+  '/wallet/',
```

Comment:
This won't match with `/wallet/send_tokens`, right?

Reply:
It won't!